content/docs/user-guide/interactivity/physics/_index.md

```diff
@@ -2,4 +2,8 @@
 title: Physics
 ---
 
-{{< placeholder >}}
+O3DE's support for physics simulation is built around:
+
+* [NIVDIA Blast](./nvidia-blast) to provide fast, high-fidelity destruction simulation.
```

Comment:
```suggestion
<!-- * [NIVDIA Blast](nvidia-blast) to provide fast, high-fidelity destruction simulation. -->
```
Blast docs are hidden for the time being.  `.\` isn't needed when stepping from index to a page in the common directory.

Reply:
Notice we have other references to blast https://o3de.org/docs/user-guide/interactivity/physics/nvidia-physx/ has links to "Simulated destruction with NVIDIA Blast" - I can clean those up if required